lib/angular/shared/geolocation/geolocation.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+/*global navigator:true*/
+
+var location = {};
+
+location.getGeolocation = function(cb) {
```

Comment:
Can you add a name to the function. I think we should avoid anonymous functions wherever possible.